plugins-ui/src/modules/plugin/models/plugin.ts

```
@@ -1,32 +1,32 @@
-    description: string;
-    metadata: {};
-    server_endpoint: string;
-    category_id: string;
-    tags: Tag[];
-    ratings: PluginRatings[];
-    pricing: PluginPricing[];
+  id: string;
+  type: string;
+  title: string;
+  description: string;
+  metadata: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)